wesnoth-did.po

```diff
@@ -518,44 +450,35 @@ msgid ""
 "ever comes of necromancy. The moment ya use it, you’ve already sold yer soul "
 "ta black magic!"
 msgstr ""
+"Že tě hanba nefackuje, Maline. I kdyby nám zajistila přežít, nikdy z "
+"nekromancie nevznikne nic dobrého. Už v okamžiku, kdy jsi ji použil, si svou "
+"duši zaprodal černé magii!"
 
 #. [message]: speaker=Malin Keshar
 #: data/campaigns/Descent_Into_Darkness/scenarios/01_Saving_Parthyn.cfg:301
-#, fuzzy
-#| msgid "What? I am hardly evil, Drogan!"
 msgid "What?! I am hardly evil, Drogan!"
-msgstr "Co? Nejsem tak zlý, Drogane!"
+msgstr "Co? Nejsem přece tak zlý, Drogane!"
```

Comment:
je to drobnost, ale zachoval bych změnu, ke které došlo v originále
Malin tak bude méně fňukat...
```suggestion
msgstr "Co?! Nejsem přece tak zlý, Drogane!"
```